Realm/Tests/RealmTests.mm

```diff
@@ -1871,6 +1871,67 @@ - (void)testWritingCopyUsesWriteTransactionInProgress
     }];
 }
 
+#pragma mark - SeedFilePath
+
+- (void)testSeedFilePath {
+    // Create a seed realm with two objects
+    RLMRealmConfiguration *seedConfig = [RLMRealmConfiguration defaultConfiguration];
+    seedConfig.fileURL = RLMTestRealmURL();
+    RLMRealm *realm = [RLMRealm realmWithConfiguration:seedConfig error:nil];
+    [realm beginWriteTransaction];
+    [IntObject createInRealm: realm withValue:@[@1]];
+    [IntObject createInRealm: realm withValue:@[@2]];
+    [realm commitWriteTransaction];
+
+    NSURL *fileURL = [NSURL fileURLWithPath:RLMRealmPathForFile(@"filename.realm")];
```

Comment:
I ran this test and I didn't see `filename.realm` at the file path.

Reply:
The removal is just extra test assurance that there is no file at the path at that point in time. The writeCopy that occurs later is expecting there to be no file. I added lines asserting this explicitly.

Reply:
I guess the assert is all that's needed. The removal is unnecessary.

Reply:
I removed the "removeItemAtPath", kept the asserts.